pydy/viz/scene.py

```diff
@@ -472,17 +472,17 @@ def display_ipython(self):
 
         self.create_static_html(silent=True)
         self._widget_dict = OrderedDict()
-        self.container = widgets.ContainerWidget()
+        self.container = widgets.Box()
         components = []
         for var, init_val in \
             zip(self.constant_variables, self.constant_values):
-            self._widget_dict[str(var)] = widgets.FloatTextWidget(value=init_val,
-                                                              description=str(var))
+            self._widget_dict[str(var)] = widgets.FloatText(value=init_val,
+                                                        description=str(var))
             components.append(self._widget_dict[str(var)])
 
-        self.button = widgets.ButtonWidget(description="Rerun Simulations")
+        self.button = widgets.Button(description="Rerun Simulations")
         def button_click(clicked):
-            self.button.add_class('disabled')
+            self.button._dom_classes = ['disabled']
```

Comment:
Is it required to use these private methods? If we can avoid using them that would be ideal because they are bound to change in IPython 4.0.